### ISSUE TITLE
График вписать в окно

### ISSUE THREAD

**Issue by GloomlySeth** [GloomlySeth/Zapravka#18]
1) Возникает впечатление, что график продолжается за пределами страницы. Поле для графика нужно заключить в границы, чтоб было понятно где оно заканчивается
2) Если сделать  временной интервал далеко вперёд или далеко назад, график никак не меняется.  Я понимаю, что он отрисовывает только кусок по которому есть точки. Но возникает вопрос при обрыве связи и отсутствии точек в течение, например, одного дня. Что будет? Он склеит 2 куска с данными? 

![default](https://user-images.githubusercontent.com/18662965/52109043-23e2b080-260d-11e9-9368-e2fe1fa22789.png)

**Comment by AlekseySmorshchok** [comment]
склеит
надо фиксить?

**Comment by GloomlySeth** [comment]
да. Нужно, чтоб время на графике отражалось адекватно. Если данных не было, чтобы пользователь видел, что в такое-то время не было данных.  И пошёл разбираться в таблицы или результаты выгрузки, почему данных не было.

**Comment by AlekseySmorshchok** [comment]
ну посмотрю что можно сделать
